code/tests/find-endpoint-r.qmd

```diff
@@ -0,0 +1,160 @@
+---
+juptyer:
+  kernels: python3
+---
+
+See https://github.com/luukvdmeer/sfnetworks/blob/main/vignettes/sfn02_preprocess_clean.Rmd and the equivalent python code.
+
+```{r}
+library(sfnetworks)
```

Comment:
This is the code I mentioned @juanfonsecaLS1, please try to reproduce and let us know thoughts.